test/library/standard/CSV/CSV.chpl

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2004-2018 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module CSV {
+  use IO;
+  use DataFrames;
+
+  // TODO: dependent on #9790 or #7134
+  /*
+  iter linesCSV(path: string) {
+    var f = try! open(path, iomode.r);
+    var lines = try! f.lines().these();
```

Comment:
This doesn't need to read the whole file at once, it could just do `for line in f.lines() do ...` to read a line at a time.  That would be better for really big files.

Reply:
I agree, but #9790 (and to a lesser extent #7134) are preventing this iterator from working.